.github/workflows/main.yml

```diff
@@ -21,6 +21,8 @@ jobs:
           pip3 install -r requirements.txt
           pip3 install pytest
       - name: Run linter
-        run: pylint3 rr_bot/**/*.py
+        run: pylint3 wheatley/**/*.py
       - name: Run tests
         run: python3 -m pytest
+      - name: Check version string consistency
+        run: ./check-version-strings
```

Comment:
How does this signal success failure? Exit codes from the file?

Reply:
Yep it exits with code `1`, which GitHub assumes is a failure.  I've deleted this, though, in favour of having a `version.txt` file written by the GitHub action which publishes the package.